### ISSUE TITLE
Add Preference for colorblind safe colors

### ISSUE THREAD

**Issue by edwardrowe** [redbluegames/unity-mulligan-renamer#114]
The Diff and operation colors could use a color blind safe palette. Note this may be moot with the resolution of Issue #101, but in the meantime I suspect it's a problem.

Use http://colororacle.org/ to help evaluate the color choices.

For implementation, I would add a flag to base Drawer to specify to draw as a ColorBlind variant. HighlightColor property then be reworked to call one of two properties, depending on their color blind setting.

Stretch goal would be to consider patterns in the operation color strips. The challenge to that is Drawers don't really support drawing anything except rectangles at the moment, which is handled in the base drawer which gets a color from inherited classes.

As far as storage of the colorblind setting, this is dependent on #206. For now the simplest solution is best. A checkbox on the window that sets and gets a value in MulliganUserPreferences would be just fine.

~This should also leverage [Unity's Preference system](https://docs.unity3d.com/ScriptReference/PreferenceItem.html)~ Edit: Best practices for user preferences for plugins has changed. This is now dependent on a robust user preferences system (https://github.com/redbluegames/unity-mulligan-renamer/issues/206).

**Comment by edwardrowe** [comment]
I'm gonna call this done with #206. The only other colors I might change are the "Add", "Modify", "Remove", etc colors, but the Ux really isn't that dependent on those colors. I suspect color blind users would have no problem using the tool, now that the Diff colors can be assigned.
